variables.tf

```diff
@@ -54,4 +64,22 @@ variable "enable_log_analytics_workspace" {
   type        = bool
   description = "Enable the creation of azurerm_log_analytics_workspace and azurerm_log_analytics_solution or not"
   default     = true
+}
+
+variable "vnet_subnet_id" {
+  description = "(Optional) The ID of a Subnet where the Kubernetes Node Pool should exist. Changing this forces a new resource to be created."
+  type        = string
+  default     = null
+}
+
+variable "os_disk_size_gb" {
+  description = "Disk size of nodes in GBs."
+  type        = number
+  default     = 50
+}
+
+variable "enable_http_application_routing" {
+  description = "Enable HTTP Application Routing Addon (forces recreation)."
+  type        = bool
+  default     = false
```

Comment:
it seems adding those extra fields is to serve another task, could we split into different PR?

Reply:
I normally to merge functions to one PR to serve one release.